physics/barycentric_rotating_dynamic_frame_test.cpp

```diff
@@ -219,26 +214,21 @@ TEST_F(BarycentricRotatingDynamicFrameTest, CoriolisAcceleration) {
        Velocity<MockFrame>({(80 - 30) * Metre / Second,
                             (-60 - 40) * Metre / Second,
                             0 * Metre / Second})};
-  DegreesOfFreedom<ICRFJ2000Equator> const big_dof =
-      {Displacement<ICRFJ2000Equator>({0.8 * Metre, -0.6 * Metre, 0 * Metre}) +
-           ICRFJ2000Equator::origin,
-       Velocity<ICRFJ2000Equator>({-16 * Metre / Second,
-                                   12 * Metre / Second,
-                                   0 * Metre / Second})};
-  DegreesOfFreedom<ICRFJ2000Equator> const small_dof =
-      {Displacement<ICRFJ2000Equator>({5 * Metre, 5 * Metre, 0 * Metre}) +
-           ICRFJ2000Equator::origin,
-       Velocity<ICRFJ2000Equator>({40 * Metre / Second,
-                                   -30 * Metre / Second,
-                                   0 * Metre / Second})};
-  DegreesOfFreedom<ICRFJ2000Equator> const barycentre_dof =
-      Barycentre<DegreesOfFreedom<ICRFJ2000Equator>, GravitationalParameter>(
+  DegreesOfFreedom<ICRS> const big_dof = {
```

Comment:
I am not in love with `{` on a line of its own.  Could we put it at the beginning of the next line?